prow/tide/tide.go

```diff
@@ -2263,3 +2250,13 @@ func mapKeyWithHighestvalue(m map[string]int) string {
 
 	return result
 }
+
+func getBetterSimpleState(a, b simpleState) simpleState {
```

Comment:
nit: would be nice to mention here that only "success, pending, failure" states are supported statues

Reply:
Updated, wdyt?